### PR TITLE
icalingua++: new, 2.6.3

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

icalingua++: new, 2.6.3

Package(s) Affected
-------------------

icalingua++: new, 2.6.3

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

### CODE REVIEW

**Comment by eatradish** [comment]
LGTM on AMD64:

image

**Comment by eatradish** [comment]
LGTM on PBP (arm64), but preference looks such a noob...

image

File: extra-web/icalingua++/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME="icalingua++"
+PKGDES="Unofficial QQ Client"
```
Comment (MingcongBai) [inline extra-web/icalingua++/autobuild/defines]: An unofficial QQ client

File: extra-web/icalingua++/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+PKGDEP="atk libappindicator"
```
Comment (MingcongBai) [inline extra-web/icalingua++/autobuild/defines]: Any GTK+ dependencies?

**Review by MingcongBai — DISMISSED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**